docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -48,6 +48,133 @@ Creating a development environment (platform-agnostic steps)
    ``$ . venv/bin/activate && . hacking/env-setup``
 
 
+Creating an info module
+=======================
+
+Ansible gather information about the target machines using facts modules.
```

Comment:
```suggestion
Ansible gather information about the target machines using info modules.
```

- would be good to put something that in general such new modules should be `_info` (have `_info` suffix) and why
- maybe it's also worth adding a note that before Ansible 2.* there were _facts modules that were renamed and using `_facts` is deprecated, etc.
- or if all info modules must have `_info` suffix without any exceptions (i'm not sure), alternatively, we could just replace all `_facts` with `_info`.